src/main/java/com/btkelly/gnag/tasks/CheckLocalTask.java

```diff
@@ -80,4 +88,683 @@ public void taskAction() {
             Logger.logError("Gnag check failed but configuration allows build success");
         }
     }
+
+    private String getGitHubStyleCustomizations() {
+        return "<style>\n"
+            + "    .markdown-body {\n"
+            + "        box-sizing: border-box;\n"
+            + "        min-width: 200px;\n"
+            + "        max-width: 980px;\n"
+            + "        margin: 0 auto;\n"
+            + "        padding: 45px;\n"
+            + "    }\n"
+            + "</style>\n";
+    }
+
+    private String getStandardGitHubStyles() {
```

Comment:
Hmm this is tough to maintain, I really think this needs to be it's own CSS file.